docs/source/markdown/podman-image-prune.1.md

```diff
@@ -11,6 +11,8 @@ podman-image-prune - Remove all unused images from the local store
 you can delete all unused images.  Unused images are dangling images as well as any image that
 does not have any containers based on it.
 
+The image prune command does not prune any cache image, i.e. intermediate images built during image creation.
```

Comment:
This seems incorrect; it _will_ prune intermediate images that are not parents of a tagged image, won’t it?

Reply:
Still outstanding.

Reply:
This is fixed. 
updated text is 
```
The image prune command does not prune any valid cache image, i.e. intermediate images built during tagged image creation.
```

Reply:
I don’t understand “valid” in that context; what would an “invalid” cache image be?

Either way the “i.e.” explanation for what the “valid cache image” means does not match what the code does.

Reply:
Writing too much explanation like explaining valid image may not look good. Will removing valid will fix it?

Reply:
No, that just removes an attempt to fix the problem.

How about something like “does not prune cache images that only use layers that are necessary for other uses”?

Reply:
Updated it.